### PR TITLE
Update changelog for v0.2.2 release and tick version

### PR DESCRIPTION
**Pull request by thenewwazoo**
Added updates to changelog (worded them as best I could based on reading PRs and code).

AFAICT there are no breaking changes here (e.g. those being discussed in #100)

### CODE REVIEW

**Comment by thenewwazoo** [comment]
Just a friendly ping to see if this can land soon. I've got a handful of HAL crates waiting on this before publishing.

**Comment by ryankurte** [comment]
Hey, thanks for the ping. My perspective is that I'd like to see adaptors between the new/old traits before we publish a new release so we don't end up with fragmentation/incompatibility between implementations of the new/old versions, and I think this is something that we need agreement on first (maybe in #100).

**Comment by ryankurte** [comment]
Ahh, sorry based on previous discussion in #92 that'll be `0.2.3`. This looks good to me.

bors r+

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,20 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [v0.2.2] - 2018-11-03
```
Comment (therealprof) [inline CHANGELOG.md]: @thenewwazoo The link definition for the changes between v0.2.1 and v0.2.2 is missing.

**Comment by therealprof** [comment]
This has been published to crates.io.

**Pull request merged**